nucypher/blockchain/eth/actors.py

```diff
@@ -286,14 +288,16 @@ def deploy_network_contracts(self,
         # deploy contracts
         total_deployment_transactions = 0
         for deployer_class in self.deployer_classes:
-            total_deployment_transactions += deployer_class.number_of_deployment_transactions
+            total_deployment_transactions += len(deployer_class.deployment_steps)
 
         first_iteration = True
-        with click.progressbar(length=total_deployment_transactions, label="Deployment progress") as bar:
+        with click.progressbar(length=total_deployment_transactions,
+                               label="Deployment progress",
+                               show_eta=False) as bar:
             bar.short_limit = 0
             for deployer_class in self.deployer_classes:
                 if interactive and not first_iteration:
```

Comment:
Since we are prompting before each contract and providing the name of the contract, do we want to prompt before the deployment of the first iteration contract "NuCypherToken"? As is, we deploy "NuCypherToken" contract without prompting, but then prompt for the others. Perhaps we don't - but just checking.

Reply:
I personally found it weird to type `"DEPLOY"`, waiting the countdown,... and then being interrupted with the requirement of pressing a key. We do however print `"Deploying NuCypherToken ..."`, so it's not like the user doesn't know what's happening.